ci-automation/vendor-testing/azure.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+# Copyright (c) 2021 The Flatcar Maintainers.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -euo pipefail
+
+set -x
+
+# Test execution script for the qemu vendor image.
+# This script is supposed to run in the SDK container.
+
+# $@ now contains tests / test patterns to run
+
+source ci-automation/vendor_test.sh
+
+board="${CIA_ARCH}-usr"
+basename="ci-${CIA_VERNUM//+/-}"
```

Comment:
```suggestion
basename="ci-${CIA_VERNUM//+/-}-${CIA_ARCH}"
```
Since the basename is used per test we should prevent resource name clashes when both architectures get tested.